tests/integration_tests.rs

```diff
@@ -272,6 +272,28 @@ fn squeeze_limit_line_numbers() {
         .stdout("   1 line 1\n   2 \n   3 \n   4 \n   5 line 5\n   6 \n   7 \n   8 \n   9 \n  10 \n  20 line 20\n  21 line 21\n  22 \n  23 \n  24 line 24\n  25 \n  26 line 26\n  27 \n  28 \n  29 \n  30 line 30\n");
 }
 
+#[test]
+fn list_themes() {
+    let default_theme_chunk: &str;
+    #[cfg(target_os = "macos")]
+    {
+        default_theme_chunk = from_utf8(b"Monokai Extended Light\x1B[0m (default)").unwrap();
+    }
+    #[cfg(not(target_os = "macos"))]
+    {
+        default_theme_chunk = from_utf8(b"Monokai Extended\x1B[0m (default)").unwrap();
+    }
```

Comment:
```suggestion
    #[cfg(target_os = "macos")]
    let default_theme_chunk = "Monokai Extended Light\x1B[0m (default)";

    #[cfg(not(target_os = "macos"))]
    let default_theme_chunk = "Monokai Extended\x1B[0m (default)";
```

Reply:
Nice idea. I didn't know block (`{}`) was optional with `#[cfg()]`. :-)
I squashed it with the commit which adds the test.